### ISSUE TITLE
Scrap Clipboard

### ISSUE THREAD

**Issue by bislara** [bislara/code-n-stitch#83]
## Description
Get all the information from the clipboard

- [ ] Store text in the clipboard
- [ ] Store image in a clipboard
- [ ] Get the recent item from the clipboard
- [ ] Get a list of the currently available types

## Technologies and Languages

- Python
- Javascript

Any other language and technologies are also fine.

## Aditional Work 
Make a GUI for this application

**Comment by Krishna-Sivakumar** [comment]
Can I try this out?

**Comment by bislara** [comment]
Sure. 
You can have this as a CLI application or you can make a GUI for this application as well .

**Comment by bislara** [comment]
@Krishna-Sivakumar  Any progress?

**Comment by Krishna-Sivakumar** [comment]
> @Krishna-Sivakumar Any progress?
yeah, I've figured out a part of it. The PR will be up pretty soon

**Comment by bislara** [comment]
Okay cool. Waiting for your PR

**Comment by bislara** [comment]
any updates @Krishna-Sivakumar ??

**Comment by Krishna-Sivakumar** [comment]
@bislara 
you can expect a PR this week.

**Comment by bislara** [comment]
Okay fine. Looking forward to it

**Comment by SouvikDcoder** [comment]
Hey, @bislara I see no update on this issue from the assigned user. Is it still open? Can I take it up?

**Comment by bislara** [comment]
Hi @SouvikDcoder, Go ahead and work on it.
